Misc/NEWS.d/next/Core and Builtins/2021-07-03-00-20-39.bpo-43908.YHuV_s.rst

```diff
@@ -0,0 +1,3 @@
+Types with the :const:`Py_TPFLAGS_IMMUTABLETYPE` flag can now inherit the
```

Comment:
I suggest to specify **heap** types:

```suggestion
Heap types with the :const:`Py_TPFLAGS_IMMUTABLETYPE` flag can now inherit the
```

Reply:
FYI, also added to the What's New entry.